packages/neutrino/index.js

```diff
@@ -31,6 +31,13 @@ module.exports = (middleware = {}) => {
     neutrino.config.mode(mode);
   }
 
+  // The default output is too noisy, particularly with multiple entrypoints.
```

Comment:
This feels like the wrong place to make this change. I know it's more duplication, but I would rather this lived in the middleware/presets instead of here.